### PR TITLE
Update screenshots for chapter 9 (QGIS Configuration)

### PR DESCRIPTION
**Pull request by havatv**
Goal:

Ticket(s): #2441

- [ ] Backport to LTR documentation is required

### CODE REVIEW

File: source/docs/user_manual/introduction/qgis_configuration.rst [hunk 1]
```diff
@@ -11,21 +11,21 @@ QGIS Configuration
 QGIS is highly configurable. Through the :menuselection:`Settings` menu, it
 provides different tools to:
 
-* |options| :guilabel:`Options...`: set global :ref:`options <gui_options>` to
-  apply in different areas of the software. These preferences are saved in the
-  active :ref:`User profile <user_profiles>` settings and applied by default
-  whenever you open a new project with this profile.
+* |styleManager| :guilabel:`Style Manager...`: create and manage :ref:`symbols and
```
Comment (DelazJ) [inline source/docs/user_manual/introduction/qgis_configuration.rst]:
```suggestion
* |styleManager| :guilabel:`Style Manager...`: create and manage :ref:`symbols,
```

File: source/docs/user_manual/introduction/qgis_configuration.rst [hunk 2]
```diff
@@ -11,21 +11,21 @@ QGIS Configuration
...
+  color ramps <vector_style_manager>`.
```
Comment (DelazJ) [inline source/docs/user_manual/introduction/qgis_configuration.rst]:
```suggestion
  styles and color ramps <vector_style_manager>`.
```

File: source/docs/user_manual/introduction/qgis_configuration.rst [hunk 3]
```diff
@@ -1211,6 +1222,13 @@ In the :guilabel:`Data Sources` tab, you can:
   * |search| :guilabel:`Filter layers...` and quickly find a particular layer to
     configure.
 
+.. _figure_datasources_tab:
+
+.. figure:: img/project_datasources.png
+   :align: center
+
+   Relations tab
```
Comment (DelazJ) [inline source/docs/user_manual/introduction/qgis_configuration.rst]:
```suggestion
   Data Sources tab
```

**Comment by havatv** [comment]
> Some remarks:
>     * resize the keyboard shortcuts screenshot/columns to show the other column
>     * the rendering menu screenshot is being updated in another PR
>     * project_general: not sure that is a normal file naming to show in docs ;)

I will try to look into this later.

**Pull request merged**